json-schema-converter/src/test/java/io/confluent/connect/json/JsonSchemaDataTest.java

```diff
@@ -261,6 +261,29 @@ public void testFromConnectNumericDecimal() {
     jsonSchemaData = new JsonSchemaData();
   }
 
+  @Test
+  public void testFromConnectNumericDecimalWithTrailingZeros() {
+    jsonSchemaData =
+        new JsonSchemaData(new JsonSchemaDataConfig(
+            Collections.singletonMap(JsonSchemaDataConfig.DECIMAL_FORMAT_CONFIG,
+                DecimalFormat.NUMERIC.name())));
+    NumberSchema schema = NumberSchema.builder()
+        .title("org.apache.kafka.connect.data.Decimal")
+        .unprocessedProperties(ImmutableMap.of("connect.type",
+            "bytes",
+            "connect.version",
+            1,
+            "connect.parameters",
+            ImmutableMap.of("scale", "4")
+        ))
+        .build();
+    checkNonObjectConversion(schema,
+        DecimalNode.valueOf(new BigDecimal("1.5600")),
+        Decimal.schema(4),
+        new BigDecimal(new BigInteger("15600"), 4)
+    );
```

Comment:
Question: why does the previous test (testFromConnectNumericDecimal) instantiate a new JsonSchemaData towards the end ?

Reply:
Good question.  That's just to clear out the JsonSchema for tests that rely on a default instance.  I'll do the same in the new test.